packages/amplify_datastore/example/lib/models/ModelProvider.dart

```diff
@@ -111,6 +120,21 @@ class ModelProvider implements ModelProviderInterface {
           return PostAuthComplex.classType;
         }
         break;
+      case "TestNullableListsEnum":
```

Comment:
I have an open PR which adds a schema.graphql file to/amplify_datastore/example/tool/. It is used as part of the provisioning script for datastore.

The new models added in this PR should probably be added to that schema so that everyone can re-generate them.